moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -44,6 +44,16 @@
 #include <boost/bind.hpp>
 #include <moveit/robot_model/aabb.h>
 
+namespace moveit
+{
+namespace core
+{
+/** \brief At least 10 steps per trajectory for testing jump thresholds with computeCartesianPath. Otherwise
+ * testJointSpaceJump doesn't work well. */
```

Comment:
expand "doesn't work well."....

"Use at least 10 steps per trajectory for testing jump thresholds with computeCartesianPath(). Otherwise testJointSpaceJump() falsely reports valid path" ?